src/components/TasksHeader/TasksHeader.tsx

```diff
@@ -31,8 +32,7 @@ export const TasksHeader = ({
 }) => {
   const { data: tasks, refetch, error } = useContext(TasksContext)
 
-  const completedTasks = tasks?.filter(task => task.status === 'COMPLETED')
-    .length
+  const completedTasks = tasks?.filter(task => isComplete(task.status)).length
```

Comment:
Many examples of this - there are multiple states we want to show as "completed" in the UI. Not intuitive, so not surprised you didn't have this info.